code/src/components/TaskCounter.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const TaskCounter = () => {
+  const todos = useSelector(store => store.tasks.todos)
+
+  return (
+    <div>
+    <div> done: {todos.filter((task) => task.complete === true).length} </div>
```

Comment:
It is interesting to see a different approach for me as you use the filter method right here.